visual-js/visual-snapshots/src/commands/options.ts

```diff
@@ -80,21 +82,21 @@ export const suiteNameOption = new Option(
   "The name of the suite you would like to appear in the Sauce Visual dashboard." +
     EOL +
     "Supports the following parameters: {filename}, {ext}, {directory}, {directoryRelative}, {page}"
-);
+).default("{directory}");
```

Comment:
Should `{directory}` be the default value here? It can be a bit unexpected IMO. Especially, when you pass multiple PDFs from different directories.

I propose leaving this undefined, or to set this to something generic, like "PDF Visual Tests".

Reply:
As agreed with the team, we will use `directoryRelative` as the default